erpnext/hr/doctype/salary_slip/salary_slip.py

```diff
@@ -204,15 +204,16 @@ def calculate_net_pay(self):
 			self.precision("net_pay") if disable_rounded_total else 0)
 
 	def on_submit(self):
-		if(self.email_check == 1):
-			self.send_mail_funct()
+		if(frappe.db.get_single_value("HR Settings", "email_salary_slip_to_employee")):
+			self.email_salary_slip()
 
 
-	def send_mail_funct(self):
-		receiver = frappe.db.get_value("Employee", self.employee, "company_email")
+	def email_salary_slip(self):
+		receiver = frappe.db.get_value("Employee", self.employee, "company_email") or \
+			frappe.db.get_value("Employee", self.employee, "personal_email")
 		if receiver:
 			subj = 'Salary Slip - ' + cstr(self.month) +'/'+cstr(self.fiscal_year)
 			frappe.sendmail([receiver], subject=subj, message = _("Please see attachment"),
-				attachments=[frappe.attach_print(self.doctype, self.name, file_name=self.name)])
+				attachments=[frappe.attach_print(self.doctype, self.name, file_name=self.name)], bulk=True)
```

Comment:
You should also use `reference_doctype` and `reference_name` to link Bulk Email to Salary Slip record